layouts/shortcodes/content-version-container.html

```diff
@@ -0,0 +1,6 @@
+<div class="content-version-container">
+
+    <label id="content-version-label" hidden>{{ with .Get "version-label" }}{{.}}{{ end }}</label>
```

Comment:
@andreasbalevik Use class instead of id to avoid having the same id multiple times in the generated HTML?

Reply:
@altinnadmin Yes, much better.